app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -100,25 +101,28 @@ fun OtherIconText() {
 
 @Composable
 fun TotalText() {
-    Text(
+    Row(verticalAlignment = Alignment.CenterVertically){Text(
         text = "Total:",
-        fontSize = (20.sp),
+        fontWeight = FontWeight.Bold,
```

Comment:
The text total  will be bolded

Reply:
That's style/design...  but doesn't necessarily impact technical debt, unless it aids accessibility.  If so, indicate how it does.